apps/store/src/blocks/ImageBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from '@emotion/styled'
+import { SbBlokData, StoryblokComponent, storyblokEditable } from '@storyblok/react'
+import Image from 'next/image'
+import { SbBaseBlockProps, StoryblokImage } from '@/services/storyblok/storyblok'
+
+type ImageBlockProps = SbBaseBlockProps<{
+  image: StoryblokImage
+  fullBleed?: boolean
+  body?: Array<SbBlokData>
+}>
+
+export const ImageBlock = ({ blok }: ImageBlockProps) => {
+  const sizeProps = getSizeFromURL(blok.image.filename)
+
+  return (
+    <Wrapper {...storyblokEditable(blok)} margins={blok.fullBleed ? false : true}>
```

Comment:
Thinking this might be a bit more easy to read.

```suggestion
    <Wrapper {...storyblokEditable(blok)} margins={!blok.fullBleed>
```